src/react-uploader/list.js

```diff
@@ -14,7 +14,11 @@ export default class List extends React.Component {
 
           <div className="upl-list-img">
             <div className="img-center">
-              <img src={item.cdnUrl || item.url} />
+              {item.isImage && (item.cdnUrl || item.url) ?
```

Comment:
The other issue thats come up with this approach is that they can upload things like pdf which are now good, should have to be a video or an image otherwise it shouldnt allow it to be uploaded at all